src/api.ts

```diff
@@ -64,6 +66,7 @@ export const apiFactory = (serviceId: number) => {
 		}
 		metrics.inc(Metrics.OnlineDevices);
 		metrics.inc(Metrics.TotalDevices);
+		workerMap[req.params.common_name] = req.params.worker;
```

Comment:
Where does `common_name` come from in the `req.params`?

Reply:
[this](https://github.com/balena-io/open-balena-vpn/blob/master/openvpn/scripts/client-connect.sh#L26-L33) is called by openvpn when a client connects.